internal/dataplane/parser/translate_ingress.go

```diff
@@ -217,6 +219,28 @@ func ingressV1ToKongServiceLegacy(
 	return servicesCache
 }
 
+// routeNamer returns function for generating a name for a Kong Route based on the Ingress name, namespace, rule index and path index.
+// If the name won't be unique, it registers a failure.
+func routeNamer(failuresCollector *failures.ResourceFailuresCollector, objIngress client.Object) func(ruleIndex, pathIndex int) *string {
+	uniqueRouteNames := make(map[string]struct{})
+	return func(ruleIndex, pathIndex int) *string {
+		routeName := kong.String(fmt.Sprintf("%s.%s.%d%d", objIngress.GetNamespace(), objIngress.GetName(), ruleIndex, pathIndex))
+		if _, conflicting := uniqueRouteNames[*routeName]; conflicting {
+			failuresCollector.PushResourceFailure(
+				fmt.Sprint(
+					"ERROR: Kong route with conflicting name: ", *routeName, " ",
```

Comment:
```suggestion
					"Kong route with conflicting name: ", *routeName, " ",
```
I think we should not add such prefixes to event messages. The severity of an event is already defined by its type.